rpc_jobs/long_running_slave.yml

```diff
@@ -0,0 +1,96 @@
+- job:
+    name: Long-Running-Slave
+    project-type: workflow
+    concurrent: true
+    properties:
+      - build-discarder:
+          num-to-keep: 30
+    parameters:
+      # See params.yml
+      - rpc_gating_params
+      - instance_params:
+          IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
+          REGION: "IAD"
+          FLAVOR: general1-8
+      - jenkins_node_params:
+          JENKINS_NODE_LABELS: "pubcloud_multiuse"
+          JENKINS_NODE_EXECUTORS: "16"
+          jenkins_node_exclusive: false
+          allow_jenkins_sudo: false
+      - rpc_maas_params:
+          RPC_MAAS_BRANCH: master
+      - string:
+          name: STAGES
+          default: "Allocate Resources, Connect Slave, Maas Agent"
+          description: |
+            Pipeline stages to run CSV. Note that this list does not influence execution order.
+            Options:
+              Allocate Resources
+              Connect Slave
+              Cleanup
+              Destroy Slave
+
+    dsl: |
+      node(){
+          dir("rpc-gating"){
+            git branch: env.RPC_GATING_BRANCH, url: env.RPC_GATING_REPO
+            common = load 'pipeline_steps/common.groovy'
+            pubCloudSlave = load 'pipeline_steps/pubcloud.groovy'
+            maas = load 'pipeline_steps/maas.groovy'
+          }
+          dir("rpc-maas"){
+            git branch: env.RPC_MAAS_BRANCH, url: env.RPC_MAAS_REPO
+          }
+          instance_name = common.gen_instance_name()
```

Comment:
They would be the only instances with the lrs prefix, so should be easy to spot. However as this has two votes, I've changed the gen_instance_name function to allow specifying a custom prefix, and have specified prefix="long-running-slave" for this job.